src/lib/ngsi/ContextAttributeVector.cpp

```diff
@@ -71,6 +71,74 @@ static std::string addedLookup(const std::vector<std::string>& added, std::strin
 
 /* ****************************************************************************
 *
+* ContextAttributeVector::renderV2 - 
+*
+* Attributes named 'id' or 'type' are not rendered in API version 2, due to the 
+* compact way in which API v2 is rendered. Attributes named 'id' or 'type' would simply
+* collide with the 'id' and 'type' of the entity itself (holder of the attribute).
+*
+* If anybody (stupid enough) needs an attribute named 'id' or 'type', then API v1
```

Comment:
I think is better (and will survive the API v1 removal :) to say: "... then standard operation API will have to be used to retreive that information".